libdnf/repo/repo_sack.cpp

```diff
@@ -351,8 +351,20 @@ void RepoSack::create_repos_from_file(const std::string & path) {
     }
 }
 
-void RepoSack::create_repos_from_config_file() {
-    create_repos_from_file(std::filesystem::path(base->get_config().config_file_path().get_value()));
+void RepoSack::create_repos_from_config_file() try {
+    const auto & path = base->get_config().config_file_path().get_value();
+    try {
+        create_repos_from_file(std::filesystem::path(path));
+    } catch (const std::filesystem::filesystem_error & e) {
+        std::throw_with_nested(InaccessibleConfigError(M_("Unable to access configuration file \"{}\""), path));
```

Comment:
Do we need these nested try/catch blocks? I mean, couldn't the `if (base->get_config().config_file_path().get_priority() >= libdnf::Option::Priority::COMMANDLINE)` condition be right here?